Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -147,21 +146,36 @@ namespace ROS2
             else if (Utils::IsFileUrdf(m_urdfPath))
             {
                 // standard URDF
-                parsedUrdfOutcome = UrdfParser::ParseFromFile(m_urdfPath, parserConfig);
+                parsedUrdfOutcome = UrdfParser::ParseFromFile(m_urdfPath, parserConfig, sdfBuilderSettings);
             }
             else
             {
                 AZ_Assert(false, "Unknown file extension : %s \n", m_urdfPath.c_str());
             }
             AZStd::string log;
             bool urdfParsedSuccess{ parsedUrdfOutcome };
+            bool urdfParsedWithWarnings{ parsedUrdfOutcome.m_modifiedURDFContent.size() > 0};
             if (urdfParsedSuccess)
             {
+                if (urdfParsedWithWarnings)
+                {
+                    report += "# " + tr("The URDF was parsed, but it was modified to be compatible") + "\n";
```

Comment:
```suggestion
                    report += "# " + tr("The URDF was parsed, though results were modified to be compatible with SDFormat") + "\n";
```

Reply:
Not sure if this is right - but please add an explanation (compatible with what?)